src/db/scripts/populate-test-db.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * populate-test-db.js
+ * 
+ * Requirements Met:
+ * - The script uses the existing MongoDB connection logic from `db/index.js`.
+ * - Data to be populated is defined as per the provided structure.
+ * - The database is cleared before inserting new data to ensure clean and consistent test data.
+ * - The script logs success and error messages on execution.
+ * - No Express server is required to run this script.
+ */
+import mongoose from 'mongoose';
+import { User } from '../../models/user.model.js'; // Import User model
+import connectDB from '../index.js'; // Import connectDB function to connect to the database
+
+const users = [
```

Comment:
Changes in data entries:
- Enter realistic entries
- It current Semester is 4, then User details property of the user must have details of all sems upto 4